package.json

```diff
@@ -25,13 +25,13 @@
 		"@reach/dialog": "0.15.0",
 		"@reach/tabs": "0.15.0",
 		"@synthetixio/assets": "2.0.10",
-		"@synthetixio/contracts-interface": "2.48.7",
-		"@synthetixio/data": "2.48.7",
-		"@synthetixio/optimism-networks": "2.48.4",
-		"@synthetixio/providers": "2.48.4",
-		"@synthetixio/queries": "2.48.8",
-		"@synthetixio/transaction-notifier": "2.48.4",
-		"@synthetixio/wei": "2.48.4",
+		"@synthetixio/contracts-interface": "2.49.0",
+		"@synthetixio/data": "2.49.0",
+		"@synthetixio/optimism-networks": "2.49.0",
```

Comment:
Upgrading `@synthetixio/optimism-networks` introduces too many bugs. Is there a reason this needs to be upgraded?